spec/all_spec.rb

```diff
@@ -1,10 +1,11 @@
 require 'spec_helper'
 
 describe "QboApi Import All entities" do
+  let(:api){QboApi.new(creds.to_h)}
```

Comment:
Style: Have a space for items inside `{}` e.g. `let(:api){ QboApi.new(creds.to_h) }`